src/vstest.console/Publisher/MetricsPublisher.cs

```diff
@@ -70,6 +75,13 @@ public void PublishMetrics(string eventName, IDictionary<string, object> metrics
                 }
 
                 this.session.PostEvent(telemetryEvent);
+
+                // Log to Text File
+                var logEnabled = Environment.GetEnvironmentVariable("VSTEST_LOGTELEMETRY");
```

Comment:
Can you add the telemetry key/value pairs in `EqtTrace` on line 63?

Reply:
Done.